packages/wallets/src/features/cashier/modules/WithdrawalCrypto/WithdrawalCrypto.tsx

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
-import { useActiveWalletAccount, useCurrencyConfig } from '@deriv/api';
+import React, { useState } from 'react';
+import { useActiveWalletAccount, useCryptoWithdrawal, useCurrencyConfig } from '@deriv/api';
 import { WalletText } from '../../../../components';
 import { TransactionStatus } from '../TransactionStatus';
-import { WithdrawalCryptoForm, WithdrawalCryptoDisclaimer } from './components';
+import { WithdrawalCryptoDisclaimer, WithdrawalCryptoForm, WithdrawalCryptoReceipt } from './components';
 import './WithdrawalCrypto.scss';
 
 type TWithdrawalCryptoProps = {
+    onClose: () => void;
     verificationCode: string;
 };
 
-const WithdrawalCrypto: React.FC<TWithdrawalCryptoProps> = ({ verificationCode }) => {
+type TWithdrawalReceipt = {
+    address: string;
+    amount: string;
+    currency?: string;
+};
+
+const WithdrawalCrypto: React.FC<TWithdrawalCryptoProps> = ({ onClose, verificationCode }) => {
     const { data: activeWallet } = useActiveWalletAccount();
+    const { isSuccess: isWithdrawalSuccess, mutateAsync } = useCryptoWithdrawal();
```

Comment:
hey @aum-deriv 
what do you think about handling withdrawal request in current component. in that case there is no need to pass `onWithdrawalSuccess` func to the child.
you can cerate a func:
```
const requestCryptoWithdrawal = (values) => {
               mutateAsync({
                    address: values.cryptoAddress,
                    amount: parseFloat(parseFloat(values.cryptoAmount).toFixed(FRACTIONAL_DIGITS_CRYPTO)),
                    verification_code: verificationCode,
                }).then(()=>setWithdrawalReceipt({ address, amount, currency }))
}
```
and pass it as a prop to the child

Reply:
This is doable! But I don't understand how will we trigger the withdrawal as it happens on submission of the form 😕. Will anyways have to pass the above as a prop to the `WithdrawalCryptoForm`.